app/models/comfy/cms/page.rb

```diff
@@ -3,14 +3,28 @@
 class Comfy::Cms::Page < ActiveRecord::Base
   scope :most_popular, -> (number_of_items) { unscoped.order('page_views desc').limit(number_of_items) }
 
-  def self.all_english_articles
+  scope :with_tags, -> (tags) { joins(:taggings).where('tag_id IN (?)', tags.map(&:id)) }
+
+  scope :remove_self_from_results, -> (page) { where.not(id: page.id) }
+
+  scope :remove_alternate_language_page_when_welsh, (lambda do |page|
+    where.not(id: page.mirrors.first.try(:id))
```

Comment:
Not working on this today I think, but I was curious so I ran something to find out:

```
> Comfy::Cms::Page.all.unscoped.where.not(id: nil)
  Comfy::Cms::Page Load (11.1ms)  SELECT `comfy_cms_pages`.* FROM `comfy_cms_pages`  WHERE (`comfy_cms_pages`.`id` IS NOT NULL)
```

Shouldn't change anything, but it is admittedly unexpected.